docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,530 @@
+targetShareBp = 10000  # 100%
+
+# Share of the rewards which goes to the staking module (in basis points, 100% = 10000)
+# # https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/StakingRouter.sol#L167-L175
+stakingModuleFee = 500  # 5%
+
+# Share of the rewards which goes to the treasury (in basis points, 100% = 10000)
+# # https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/StakingRouter.sol#L167-L175
+treasuryFee = 500  # 5%
+
+stakingModuleId = 1
+stakingModuleName = "curated-onchain-v1"
+stakingModuleType = 0x637572617465642d6f6e636861696e2d76310000000000000000000000000000  # bytes32("curated-onchain-v1")
+```
+
+## OracleDaemonConfig
+
+```python
+# Parameters realted to "bunker mode"
+# See https://research.lido.fi/t/withdrawals-for-lido-on-ethereum-bunker-mode-design-and-implementation/3890
+# BASE_REWARD_FACTOR: https://ethereum.github.io/consensus-specs/specs/phase0/beacon-chain/#rewards-and-penalties
+NORMALIZED_CL_REWARD_PER_EPOCH=64
+NORMALIZED_CL_REWARD_MISTAKE_RATE_BP=1000  # 10%
+REBASE_CHECK_NEAREST_EPOCH_DISTANCE=1
+REBASE_CHECK_DISTANT_EPOCH_DISTANCE=23  # 10% of AO 225 epochs frame
+
+# See https://snapshot.org/#/lido-snapshot.eth/proposal/0xa4eb1220a15d46a1825d5a0f44de1b34644d4aa6bb95f910b86b29bb7654e330
+VALIDATOR_DELAYED_TIMEOUT_IN_SLOTS=7200  # 1 day
+VALIDATOR_DELINQUENT_TIMEOUT_IN_SLOTS=28800  # 4 days
+NODE_OPERATOR_NETWORK_PENETRATION_THRESHOLD_BP=100  # 1% network penetration for a single NO
+
+# Time period of historical observations used for prediction of the rewards amount 
+PREDICTION_DURATION_IN_SLOTS=50400  # 7 days
+
+# Max period of delay for requests finalization in case of bunker due to negative rebase
+# twice min governance response time - 3 days voting duration
+FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT=1350  # 6 days
+```
+
+## OracleReportSanityChecker
+
+```python
+# Sanity limit on the number of deposits: not more than ~half of the current DSM deposits capacity (43200 it is)
+# https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L221-L232
+churnValidatorsPerDayLimit = 20000
+
+# Taken from current oracle limit https://etherscan.io/address/0x442af784A788A5bd6F42A01Ebe9F287a871243fb#readProxyContract#F8
+# See https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L47-L50
+oneOffCLBalanceDecreaseBPLimit = 500  # 5%
+
+# See https://research.lido.fi/t/increasing-max-apr-sanity-check-for-oracle-lido-report/3205
+# Related to Consensus Layer rewards only
+annualBalanceIncreaseBPLimit = 1000  # 10%
+
+# See https://github.com/lidofinance/lido-dao/blob/feature/shapella-upgrade/contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol#L647-L672
+simulatedShareRateDeviationBPLimit = 50  # 0.5%
```

Comment:
```
Staking rate limit = 150 000 
TVL = 3 000 000
⇒
SRL = 0.1
L = (2 * SRL) * max(oneOffCLBalanceDecreaseBPLimit, maxPositiveTokenRebase)
L = 50
```

Shall we introduce a staking rate limit to have 1:1 mapping with config and to support this formula?
I think so, but up to you :pray: